docs/source/markdown/podman-image-diff.1.md

```diff
@@ -1,30 +1,45 @@
 % podman-image-diff(1)
 
 ## NAME
+
 podman-image-diff - Inspect changes on an image's filesystem
 
 ## SYNOPSIS
+
 **podman image diff** [*options*] *name*
 
 ## DESCRIPTION
-Displays changes on a container or image's filesystem.  The container or image will be compared to its parent layer
+
+**podman image diff** displays changes on an image's filesystem.  The image will be compared to its parent layer.
+
+The output is prefixed with the following symbols:
+
+| Symbol | Description |
+|--------|-------------|
+| A | A file or directory was added. |
+| D | A file or directory was deleted. |
+| C | A file or directory was changed. |
 
 ## OPTIONS
 
-#### **--format**
+#### **--format**, **-f**=*format*
 
-Alter the output into a different format.  The only valid format for diff is `json`.
+Alter the output into a different format.  The only valid format for diff is `JSON`.
```

Comment:
JSON should be lowercase. The uppercase one throws an error.

Reply:
Where is the error coming from on JSON vs json?  We need to stop that error, JSON is the correct capitalization.

Reply:
It has to be fixed in containers/common https://github.com/containers/common/blob/6c933f226ed44491d082d6d52cbe31ae6fe0e76f/pkg/report/validate.go#L5 where it should be probably (json|JSON)